Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Templates/APIGatewaySetupParameters.cs

```diff
@@ -407,14 +407,15 @@ public virtual string TransformText()
             
             #line default
             #line hidden
-            this.Write("?.ContainsKey(\"");
+            this.Write("?.Any(x => string.Equals(x.Key, \"");
             
             #line 185 "C:\codebase\aws-lambda-dotnet\Libraries\src\Amazon.Lambda.Annotations.SourceGenerator\Templates\APIGatewaySetupParameters.tt"
             this.Write(this.ToStringHelper.ToStringWithCulture(headerKey));
             
             #line default
             #line hidden
-            this.Write("\") == true)\r\n            {\r\n                ");
+            this.Write("\", StringComparison.OrdinalIgnoreCase)) == true)\r\n            {\r\n                " +
```

Comment:
nit: we don't need "== true"

Reply:
This is needed by cause the `?` is used on the left side of the comparison resulting in a nullable Boolean. You can't do `if(nullable<Boolean>)` so the easy work around is to compare the nullable Boolean to `true`.